_integration-schemas/trello/boards.md

```diff
@@ -0,0 +1,264 @@
+---
+tap: "trello"
+version: "1"
+key: ""
+
+name: "boards"
+doc-link: "https://developer.atlassian.com/cloud/trello/rest/#api-boards-id-get"
+singer-schema: "https://github.com/singer-io/tap-trello/blob/master/tap_trello/schemas/boards.json"
+description: |
+  The {{ table.name }} table contains information about your boards idividual boards in {{ integration.display_name }}.
```

Comment:
I'd add a note here that this table supports custom fields - a lot of folks have been asking for this, so calling it out will be helpful: https://github.com/singer-io/tap-trello/blob/master/tap_trello/streams.py#L234